components/brave_new_tab_ui/helpers/cryptoDotComApiHelpers.ts

```diff
@@ -0,0 +1,51 @@
+function getCryptoDotComTickerInfo (asset: string) {
```

Comment:
Seems like this module should actually be located at `components/brave_new_tab_ui/api/cryptoDotCom.ts`

Reply:
(and missing license)

Reply:
moved and added license